### ISSUE TITLE
The Encoder configurations do not have a config for 1080P MP4

### ISSUE THREAD

**Issue by walt93** [WWBN/AVideo#9650]
**Is your feature request related to a problem? Please describe.**
I am trying to do some things with ffmpeg, and need to change the ffmpeg configuration in the encoder. There is no 1080P setting.

**Describe the solution you'd like**
Add the 1080P setting to the Encoder configurations.

**Comment by DanielnetoDotCom** [comment]
No, this is only available on automatic resolutions, you may need to read this to understand better

https://github.com/WWBN/AVideo/wiki/About-Video-Upload

**Comment by walt93** [comment]
Daniel, it appears that I did not succeed in communicating the issue to you.

I want to EDIT THE FFMPEG CONFIGURATION. Perhaps to change bit rate. Maybe I want to add a watermark. Does not matter. I want to change the parameters.

1) There is no way to do this. You edit the FFMPEG configuration, run a transcode, do a "ps -ax | grep ffmpeg" and find that the ffmpeg is not using the parameters you set.

2) If you did add 1080P parameters to this window, you would have to also use these parameters. I scanned the encoder source code last night. It appears to hard code parameters. I could not find code that reads from this UI, e.g. this UI appears to be placebo.

3) So there is no way to edit ffmpeg configuration AND there is no way to edit ffmpeg 1080P configuration.

I hope that clarifies things. If you are still not understanding my question please ask a LLM to translate to your native tongue. :-)

**Comment by JoshWho** [comment]
You can add a 1080p you just need to enable it in the custom advance and add it to your encoder servers.

**Comment by walt93** [comment]
![Screenshot 2024-12-03 at 11 57 53 PM](https://github.com/user-attachments/assets/648c41bc-eef1-4de3-93a9-6bf076052f1d)

This is where there is no 1080P configuration.

I want to set custom encoder parameters for 1080P. But they are not here.

See also https://github.com/WWBN/AVideo/issues/9649 where the encoder does not respect the parameters the User has set on this screen

**Comment by DanielnetoDotCom** [comment]
You can change the HD to be 1080 instead of 720

**Comment by walt93** [comment]
Now see the other bug report I posted where changing these parameters has no effect.

We can type anything we want into these fields, apparently, and the encoder does not use them.

**Comment by DanielnetoDotCom** [comment]
If you are using automatic yes it does not use them

But it depends on a lot what you setup in the customize advanced plugin

**Comment by DanielnetoDotCom** [comment]
https://github.com/WWBN/AVideo/wiki/About-Video-Upload#disabling-automatic-resolutions
